### DISCUSSION TITLE
Issue about active power output of ESD1s

### DISCUSSION THREAD

**Discussion by ErnestoHrs** [CURENT/andes#540] [Q&A] [answered]
Dear Andes developers,
In Andes, the initial reference power of energy storage is equal to gamma times the power of static PV. Does this mean that the initial state of energy storage can only be discharge? How to realize that energy storage is in a charging state from the beginning?

**Comment by jinningwang** [comment, marked as answer]
Change the corresponding PV.p0 to negative values should work.
